addons/survey/models/survey_survey.py

```diff
@@ -434,6 +498,71 @@ def _get_survey_questions(self, answer=None, page_id=None, question_id=None):
             questions = questions & answer.predefined_question_ids
         return questions, page_or_question_id
 
+    # ------------------------------------------------------------
+    # SESSIONS MANAGEMENT
+    # ------------------------------------------------------------
+
+    def next_question(self):
```

Comment:
rename in `go_to_next_question` or `session_next_question` or `start_next_question`